std/socket.d

```diff
@@ -629,48 +629,18 @@ private mixin template socketOSExceptionCtors()
 
 
 /**
- * Class for exceptions thrown from an $(D InternetHost).
+ * Class for exceptions thrown from an `InternetHost`.
  */
 class HostException: SocketOSException
 {
     mixin socketOSExceptionCtors;
 }
 
 /**
- * $(D InternetHost) is a class for resolving IPv4 addresses.
+ * `InternetHost` is a class for resolving IPv4 addresses.
  *
- * Consider using $(D getAddress), $(D parseAddress) and $(D Address) methods
+ * Consider using `getAddress`, `parseAddress` and `Address` methods
  * instead of using this class directly.
- *
- * Example:
```

Comment:
Removed Example from ddoc and just made the unittest part of the docs, because:

- it was doing the same thing
- in case of a change in the API, we do not need to worry about modifying the example as well (that is why we have `///unittest`)